src/widget/sidebar.c

```diff
@@ -329,30 +334,53 @@ static void draw_extra_info_objective(int x_offset, int y_offset, int text_group
     text_draw_number(obj->target, '(', ")", x_offset + 11 + width, y_offset + 16, font);
 }
 
+
 static void draw_extra_info_panel(int x_offset, int extra_info_height)
 {
     int y_offset = FILLER_Y_OFFSET;
 
+    // Borders of the extra_info panel - extends to bottom of screen
     int panel_blocks = extra_info_height / 16;
     graphics_draw_vertical_line(x_offset, y_offset, y_offset + extra_info_height, COLOR_WHITE);
     graphics_draw_vertical_line(x_offset + SIDEBAR_EXPANDED_WIDTH - 1, y_offset, y_offset + extra_info_height, COLOR_SIDEBAR);
     inner_panel_draw(x_offset + 1, y_offset, SIDEBAR_EXPANDED_WIDTH / 16, panel_blocks);
 
-    lang_text_draw(45, 2, x_offset + 11, y_offset + 10, FONT_NORMAL_WHITE);
-    text_draw_percentage(data.extra_info.game_speed, x_offset + 60, y_offset + 36, FONT_NORMAL_GREEN);
-    arrow_buttons_draw(x_offset, y_offset, arrow_buttons_speed, 2);
+    int y_current_line = y_offset + EXTRA_INFO_TOP_PADDING;
+
+    // Game speed modifier in extra_info panel
```

Comment:
I don't think the comments really add something to the code.

Reply:
yeah, i put those comments in because when i was looking through this function last night, it took a bit of trial and error to determine which calls were impacting what parts of the panel.

also, while they're minor comments, they should help in the future with organization.  for example, if the game speed modifier part was moved into a separate function or wrapped in a config check, the comments i put in help organize which lines are required to make sure it all works properly.